libs/fluid-elements/checkbox/src/lib/checkbox.ts

```diff
@@ -99,15 +87,15 @@ export class FluidCheckbox extends LitElement {
       }
 
       .fluid-checkbox {
-        margin-left: -${unsafeCSS(FLUID_SPACING_2X_SMALL)};
+        margin-left: calc(var(--fluid-spacing-2x-small) * -1)
```

Comment:
If this gets annoying, we could create negative spacing tokens like `fluid-spacing-2x-small-negative` since we can generate tokens dynamically now.